object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -171,11 +170,7 @@ boolean returnsThis(final MethodInvocationTree tree) {
         || hasOldReturnsReceiverAnnotation(tree);
   }
 
-  /**
-   * Continue to trust but not check the old {@link
-   * org.checkerframework.checker.builder.qual.ReturnsReceiver} annotation, for
-   * backwards-compatibility.
-   */
+  /** Continue to trust but not check the old */
```

Comment:
Looks like we lost some of the old documentation comment?  Can we restore?

Reply:
Restored it. Now it's linked to the new location of ReturnsReceiver annotation.